rxnetty/src/main/java/io/reactivex/netty/pipeline/PrimitiveConversionHandler.java

```diff
@@ -0,0 +1,34 @@
+package io.reactivex.netty.pipeline;
+
+import io.netty.buffer.ByteBufAllocator;
+import io.netty.channel.ChannelHandler.Sharable;
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.channel.ChannelOutboundHandlerAdapter;
+import io.netty.channel.ChannelPromise;
+
+@Sharable
+public class PrimitiveConversionHandler extends ChannelOutboundHandlerAdapter {
+
+    public static final PrimitiveConversionHandler INSTANCE = new PrimitiveConversionHandler();
+
+    @Override
+    public void write(ChannelHandlerContext ctx, Object msg, ChannelPromise promise) throws Exception {
+        Object msgToWrite = msg;
+
+        if (msg instanceof String) {
```

Comment:
curious.. should this be using ByteBufUtil?

https://github.com/netty/netty/blob/master/buffer/src/main/java/io/netty/buffer/ByteBufUtil.java#L441